spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/RuleInterpreter.scala

```diff
@@ -76,6 +80,46 @@ trait RuleInterpreter {
     (intTry orElse longTry orElse doubleTry orElse boolTry) getOrElse lit(input)
   }
 
+  /**
+   * Preforms a simple type cast to input base on the DataType
+   * @param input Value to be casted
+   * @param dataType DataType of the value to be casted to
+   * @return Returns Column representation of the newly casted value
+   */
+  def simpleLiteralCast(input: String, dataType: DataType): Column = {
+    Try({
+      dataType match {
+        case _: ByteType =>
+          lit(input.toByte)
```

Comment:
Mind the under-/overflow in the integer types.

Reply:
Again what do you mean? I should be doing checks for under/overflows?